roles/kubernetes/preinstall/tasks/0040-set_facts.yml

```diff
@@ -6,14 +6,6 @@
   tags:
     - facts
 
-- name: Set os_family fact for Kylin Linux Advanced Server and openEuler
```

Comment:
Why did you move those in swapoff? Souldn't you just add your debian family extension in this file as well?

Reply:
@MrFreezeex  😊Since there is [an if logic for the ansible_os_family variable](https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/preinstall/tasks/0020-verify-settings.yml#L106-L110) in verify-settings.yml, the set_fact must be executed earlier, or can I put it at the beginning of verify-settings.yml?

Reply:
Ah yes indeed, should we just switch between verify settings and set facts so that set facts is first and verify settings second. WDYT?

Reply:
Thanks for the heads up, that is indeed a good idea.👍